### PR TITLE
More comfortable knockout

### PR DESCRIPTION
**Pull request by rocketkai**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,569 @@
...
+var addNumToRunningScore = function (roundScore) {
+  // Get hold of the current players score, and add the roundScore to it
+  currRoundParticipants[currPlayer].score += roundScore;
+};
+
+var createLeaderBoardOutput = function () {
+  // set the preamble and assign it to a variable that will eventually be returned from this function
+  var leaderBoardOutput = 'Leaderboard: ';
+
+  // As per the game instructions, feel free to display the leaderboard in any order.
```
Comment (kai-rocket) [inline script.js:202]: This is nice, might be cleaner and more idiomatic to use `/* */` (single star) multi-line comment syntax for inline multi-line comments. thanks for explaining!

File: script.js [hunk 2]
```diff
@@ -1,4 +1,569 @@
...
+      currRoundParticipants = [getPlayerRandomly(), getPlayerRandomly()];
+    }
+    // Else, we are in this mode because a player has won a previous round. add one other player to currRoundParticipants
+    else {
+      currRoundParticipants.push(getPlayerRandomly());
+    }
+    // Progress the game by changing to the next mode
+    gameMode = GAME_MODE_DICE_ROLL;
+
+    // Prompt users on who will be playing in this round
```
Comment (kai-rocket) [inline script.js:459]: Where is this comment attached to? Seems dangling. If attached to the code below, no need newline between comment and code

File: script.js [hunk 3]
```diff
@@ -1,4 +1,569 @@
...
+};
+
+/*====================================
+==============MAIN====================
+======================================*/
+/**
+ * Play Beat with variable number of players and auto-choosing the largest number from the die rolled.
+ * https://swe101.rocketacademy.co/projects/project-2-dice
+ * @param {string} input
+ */
```
Comment (kai-rocket) [inline script.js:361]: Main function is about 100 lines too long. Any way to decompose further?

File: script.js [hunk 4]
```diff
@@ -1,4 +1,569 @@
...
+  // return the loser's details
+  return loser;
+};
+
+/*====================================
+==============MAIN====================
+======================================*/
+/**
+ * Play Beat with variable number of players and auto-choosing the largest number from the die rolled.
+ * https://swe101.rocketacademy.co/projects/project-2-dice
```
Comment (kai-rocket) [inline script.js:359]: Could we update the URL link to the bootcamp.rocketacademy.co course URL? I believe this one won't work

**Review by rocketkai — COMMENTED** [review]
@jeremylim91 @awongh

**Pull request closed without merging**